ballerina/tests/test.bal

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/time;
+import ballerina/os;
+
+configurable boolean isLiveServer = os:getEnv("IS_LIVE_SERVER") == "true";
+configurable string userId = isLiveServer ? os:getEnv("TWITTER_USER_ID") : "test";
+configurable string token = isLiveServer ? os:getEnv("TWITTER_TOKEN") : "test";
+configurable string serviceUrl = isLiveServer ? "https://api.twitter.com/2" : "http://localhost:9090";
+
+ConnectionConfig config = {auth: {token: token}};
+final Client twitter = check new Client(config, serviceUrl);
+
+@test:Config {
+}
```

Comment:
We need to define test groups as follows, incase there are test cases which is supported only in one mode. Since all the existing tests are compatible with both live and mock options, we can add the same configs for all the test cases.

```suggestion
@test:Config {
    groups: ["live_tests", "mock_tests"]
}
```

Reply:
Solved in: [Update mock_service and test ballerina files ](https://github.com/ballerina-platform/module-ballerinax-twitter/pull/112/commits/22f461d825fe3e85dbd202c29aa16895feb7f68e)